zmon_worker_monitor/zmon_worker/notifications/push.py

```diff
@@ -13,9 +13,9 @@ def formatEntity(entity_id):
         acc = parts[1].split(":")
 
         if acc[0] == "aws":
-            return parts[0]+":"+acc[1][-3:]
+            return parts[0] + ":" + acc[1][-3:]
```

Comment:
I would use ``.format`` here as well. Also to be consistent with the rest of the code.